scipy/stats/_fit.py

```diff
@@ -112,11 +112,11 @@ def nllf(self, params=None, data=None):
     def plot(self, ax=None, *, plot_type="hist"):
         """Visually compare the data against the fitted distribution.
 
-        Available only if ``matplotlib`` is installed.
+        Available only if `matplotlib` is installed.
 
         Parameters
         ----------
-        ax : matplotlib.axes.Axes
+        ax : `matplotlib.axes.Axes`
```

Comment:
Do you know why we need to do that? On another project I don't have that and it does the linking properly.

Reply:
I don't know, but it was not rendering properly before.
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats._result_classes.FitResult.plot.html#scipy.stats._result_classes.FitResult.plot